### ISSUE TITLE
Error shows that ViT object has no attribute 'pool'

### ISSUE THREAD

**Issue by Abhranta** [lucidrains/vit-pytorch#58]
I am trying to load my saved model and then test it on some images. 
This is my code snippet:
device = "cuda"

with torch.no_grad():
  epoch_val_accuracy = 0
  epoch_val_loss = 0
  for data, label in val_loader:
    data = data.to(device , dtype = torch.float)
    label = label.to(device , dtype = torch.long)

    val_output = model(data)
    val_loss = criterion(val_output, label)

    acc = (val_output.argmax(dim=1) == label).float().mean()
    epoch_val_accuracy += acc / len(val_loader)
    epoch_val_loss += val_loss / len(val_loader)

the screenshot is attached 
![Screenshot from 2021-01-08 19-44-07](https://user-images.githubusercontent.com/67365559/104025030-0996b180-51ea-11eb-9bfc-33aa523ff2ea.png)

**Comment by lucidrains** [comment]
@Abhranta ohh, you may have to find the older version if the package, save the state dict, then load with the newer version of ViT so you have the pool variable

**Comment by Abhranta** [comment]
@lucidrains I am using torch.save .

**Comment by Abhranta** [comment]
@lucidrains , does this implementatin of vit comes with imagenet pretrained weights ? or are the weights randomly initialized ?

**Comment by lucidrains** [comment]
@Abhranta hmm, i'm not sure why you are missing the pool attribute, it should have been set here https://github.com/lucidrains/vit-pytorch/blob/main/vit_pytorch/vit_pytorch.py#L104

this repository doesn't contain the pretrained weights. i have a link in the readme to Ross' pytorch vision repository, which does have pretrained weights!
